src/store.cc

```diff
@@ -1136,11 +1136,8 @@ StoreEntry::abort()
     if (mem_obj->abort.callback) {
         if (!cbdataReferenceValid(mem_obj->abort.data))
             debugs(20, DBG_IMPORTANT,HERE << "queueing event when abort.data is not valid");
-        eventAdd("mem_obj->abort.callback",
-                 mem_obj->abort.callback,
-                 mem_obj->abort.data,
-                 0.0,
-                 true);
+        auto *event = eventAdd("mem_obj->abort.callback", mem_obj->abort.callback, mem_obj->abort.data, 0.0, true);
```

Comment:
This abort notification clearly should be a basic AsyncCall (with the right code context set), not a time-based event. The existing cbdataDialer() may be of help here.

----

Please note that the official call looks bad and might even be slightly buggy (and you are preserving all that): The boolean parameter must be the sixth parameter. It is fifth (i.e. weight) parameter here. I will save time by _not_ discussing the correct value for the weight parameter because the best fix is to replace eventAdd() with an AsyncCall.